src/app/(with-navbar)/components/[type]/page.tsx

```diff
@@ -55,7 +55,11 @@ export default async function ComponentPage({
       </h1>
       <div className="mt-8 space-y-16 md:mt-16">
         {component.examples.map(({ ...example }, index) => (
-          <ComponentExample key={index} {...example} />
+          <ComponentExample
+            key={index}
+            {...example}
+            height={type === "badges" ? 120 : 350}
+          />
```

Comment:
_Food for thought_: Ideal would be to have this dynamically set to the iframe's content height, much like they do in TailwindUI: [Newsletter Sections](https://tailwindui.com/components/marketing/sections/newsletter-sections). Notice how each of the components have different heights.

This is not a bad solution though. Although, we would probably have to create mappings for all of the components. 

Let's move forward for now and if you need to do this again in future PRs, let's extract this logic to make it cleaner and easier to maintain.

Reply:
Good idea, I will work on this and include it in the next PR (the one where I implement Buttons).